tests/ApiManifest.Tests/ApiManifest.Tests.csproj

```diff
@@ -1,29 +1,32 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-
+    <SignAssembly>True</SignAssembly>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
+    <AssemblyName>Microsoft.OpenApi.ApiManifest.Tests</AssemblyName>
+    <RootNamespace>Microsoft.OpenApi.ApiManifest.Tests</RootNamespace>
+    <AssemblyOriginatorKeyFile>..\..\src\sgKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="xunit" Version="2.5.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.1">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.1" />
```

Comment:
why are we downgrading dependencies here? revert

Reply:
That's odd. It looks like new > test project installed v17.7.1. I'll revert.